REMOVES COATS OF IGNORANCE

JJRcop opened the pull request:
You know I always see these damn southerners wearing those jackets, when it's a perfectly comfortable 38℉. I could almost take my clothes off.

(Adds kinda-feature-freeze to CONTRIBUTING.md)

JJRcop committed:
REMOVES COATS OF IGNORANCE

You know I always see these damn southerners wearing those jackets, when it's a perfectly comfortable 38℉. I could almost take my clothes off.

(Adds kinda-feature-freeze to CONTRIBUTING.md)

ike709 commented:
Please add that the bugfix should be linked in the OP

RemieRichards commented:
You should mention that the tokens are per FIX not per FIX PR.
so a Fix PR with 5 fixes, is 5 tokens.

JJRcop committed:
Try our new two for one package

(Elaborates about the merge tokens, PRs count towards as many tokens as bugs they fix)
